docs/src/components/codes/FootnoteRefs.jsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: CC-BY-NC-SA-4.0
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import FootnoteRef from "./FootnoteRef";
+
+function joinReactElements(arr, delimiter) {
+  return arr.reduce((acc, fragment) => {
+    if (acc === null) {
+      return fragment;
+    }
+    return (
+      <>
+        {acc}
+        {delimiter}
+        {fragment}
+      </>
+    );
+  }, null);
+}
```

Comment:
I'm curious why this is needed?

Reply:
This is to handle multiple footnotes (an array).  It produces a comma delimited list of footnote references.  This seemed like the neatest way to approach it using react elements/fragments.  I welcome a better approach.